types/sass/index.d.ts

```diff
@@ -153,12 +153,18 @@ export interface Options {
     logger?: Logger;
 }
 
+export interface LoggerOptions {
+    deprecation: boolean;
+    span?: SourceSpan;
+    stack?: string;
+}
+
 export interface Logger {
     /** This method is called when Sass emits a debug message due to a @debug rule. */
-    debug?(message: string, options: { deprecation: boolean; span?: SourceSpan; stack?: string }): void;
+    debug?(message: string, options: LoggerOptions): void;
```

Comment:
Based on the docs this is not valid see: 
https://sass-lang.com/documentation/js-api/interfaces/Logger#debug

Reply:
```suggestion
    debug?(message: string, options: {span: SourceSpan}): void;
```

Reply:
Ah, so the options types were swapped then. Fixed it as per your suggestion, but formatted using Prettier. :)